### PR TITLE
:star2:  Change UI!

### PR DESCRIPTION
**Pull request by sina-yeganeh**
Please watch demo.gif for what did I change, thank u :heart:

### CODE REVIEW

**Comment by hadizz** [comment]
Hi Sina and thanks for your great design!😍

Maybe in the future, I will redesign the extension due to some new features but I appreciate your help❤️

File: manifest.json [hunk 1]
```diff
@@ -2,20 +2,8 @@
...
-      "32": "/icons/32.png",
-      "48": "/icons/48.png",
-      "128": "/icons/128.png"
-    }
-  },
-  "icons": {
-    "16": "/icons/16.png",
-    "32": "/icons/32.png",
-    "48": "/icons/48.png",
-    "128": "/icons/128.png"
```
Comment (hadizz) [inline manifest.json:19]: why did you remove icons?

Comment (hadizz) [reply manifest.json:19]: and dump version to 2?

**Review by hadizz — COMMENTED** [review]
Please check.

File: README.md [hunk 2]
```diff
@@ -1,8 +1,15 @@
-# Persian Typo Mistake 🇮🇷
-
-[![MIT License](https://img.shields.io/apm/l/atomic-design-ui.svg?)](https://github.com/tterb/atomic-design-ui/blob/master/LICENSEs)
-
-<img src="./screenshot.png" width="420px" height="100%" alt="persian mistake typo screenshot" title="persian mistake typo screenshot"/>
+<h1 align="center">Persian Typo Mistake 🇮🇷</h1>
+
+<p align="center">
+    <img alt="GitHub issues" src="https://img.shields.io/github/issues/hadizz/persian-typo-mistake-extension?style=for-the-badge">
```
Comment (hadizz) [inline README.md:4]: Sorry, but I don't like the style you gave to the badges :) I prefer the flat style.

example: GitHub issues

Comment (hadizz) [reply README.md:4]: but like I said, great job with README too! I appreciate it.

**Comment by hadizz** [comment]
I've been waiting for your response. Now is the time to merge this pull request!

**Comment by sina-yeganeh** [comment]
Hi, sorry for my late respone :_), I couldn't check my email and thank u for your merge! :+1:
